### ISSUE TITLE
Execute multiple instances of TessBaseApi in parallel

### ISSUE THREAD

**Issue by patrick-frank-bsi** [bytedeco/javacpp-presets#997]
I want to use multiple instances of Tesseract in parallel to increase throughput.

My approach was to create new instances of  `TessBaseApi` in different Threads. However if I invoke methods of `TessBaseApi` the JVM crashes in the native C code with an EXCEPTION_ACCESS_VIOLATION (0xc0000005) at pc=0x0000000065450dec, pid=21468, tid=14540.

Is there a way that would allow me to concurrently run multiple instances of `TessBaseApi` in the same JVM?

**Comment by saudet** [comment]
Duplicate of #996
